docs/decimals.md

```diff
@@ -0,0 +1,12 @@
+# Decimals value cross chain conversion
+## Abstract
+There is a possible situation where a token is bridged between multiple chains where each chain has different decimals value and isn't aware of the different value on the opposite chain. This would result in more or less tokens being bridged from source to destination chain depending on the decimals value difference between them. Because of that we need to account for decimals value difference and keep track of decimals value conversion.
```

Comment:
My suggestion for this sentence (for me, this more or less on the first read is confusing):

original:
`This would result in more or less tokens being bridged from source to destination chain depending on the decimals value difference between them.`

suggestion:
`This could lead to either an excess or shortfall of tokens bridged from the source to the destination chain, depending on the discrepancy in decimal values between them.`